littlechef/lib.py

```diff
@@ -278,7 +278,8 @@ def get_recipes_in_cookbook(name):
     path = None
     cookbook_exists = False
     metadata_exists = False
-    for cookbook_path in cookbook_paths:
+
+    for cookbook_path in env.cookbook_search_paths:
```

Comment:
When I wrote this code I thought that it would be better to have one main arrays for all cookbook paths where others will be merged, because cookbook_paths were user customizable.

So I use cookbook_paths as place where to look for non berkshelf cookbooks and berksfile_cookbooks_directory as a place where I will berks vendor cookbooks. Then I will add both directories to one array and use them later.